server/reload.go

```diff
@@ -462,6 +471,17 @@ func (c *clientAdvertiseOption) Apply(server *Server) {
 	server.Noticef("Reload: client_advertise = %s", c.newValue)
 }
 
+// accountsOption implements the option interface.
+// Ensure that authorization code is executed if any change in accounts
+type accountsOption struct {
```

Comment:
Can we just new alias feature since we are 1.9+?

Reply:
No because we would then have compilation errors when trying to define the specific Apply() functions (redeclaration).